packages/sitecore-jss/src/site/site-resolver.ts

```diff
@@ -0,0 +1,46 @@
+import { SiteInfo } from './graphql-siteinfo-service';
+
+/**
+ * Information about the current host
+ */
+export type HostInfo = {
+  hostName: string;
+  language?: string;
+};
+
+// Delimiters for multi-value hostnames
+const DELIMITERS = /\|/g;
+
+// Wildcard hostname
+const WILDCARD = '*';
+
+/**
+ * Determines site name based on the provided hostname
+ */
+export class SiteResolver {
+  /**
+   * Resolve siteName by hostName
```

Comment:
Same, update comment
```suggestion
   * Resolve siteName by host information
```